.github/mergify.yml

```diff
@@ -58,26 +143,307 @@ pull_request_rules:
 
   - name: "ask to resolve conflict"
     conditions:
+      - and: *is_open
       - "conflict"
     actions:
       comment:
         message: "This pull request now has conflicts. Could you fix it @{{author}}? 🙏"
+      label:
+        toggle:
+          - conflict
+
+  - name: "ask to not edit the readme"
+    conditions:
+      - and: *is_open
+      - files=README.md
+      - -files=README.yaml
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > Do not edit the `README.md` directly. It's auto-generated from the `README.yaml`
+          >
+
+          Rebuild the `README.md` by running `make readme` and commit the changes.
+
+          ```shell
+          make init
+          make readme
+          ```
+
+          Could you fix it @{{author}}? 🙏
+
+  - name: "ask to rebuild readme"
+    conditions:
+      - and: *is_open
+      - files=README.yaml
+      - -files=README.md
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > `README.md` is out of date.
+          >
+
+          Rebuild the `README.md` by running `make readme` and commit the changes.
+
+          ```shell
+          make init
+          make readme
+          ```
+
+          Could you fix it @{{author}}? 🙏
+
+  - name: "ask for title"
+    conditions:
+      - and: *is_open
+      - -title~=^[0-9A-Za-z]+
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > ## Title is necessary and should not be empty.
+          >
+          > Kindly provide a meaningful title for this Pull Request.
+
+  - name: "ask for description"
+    conditions:
+      - and: *is_open
+      - -body~=[0-9A-Za-z]{10,}
```

Comment:
This matches when the body doesn't contain at least one 10 char word. Is that to look for the `references` heading?
We could also check that there is at least one line with 20 common chars including whitespace, e.g. `^[0-9A-Za-z\t \-,\/\\]{20,}` and also check for consecutive whitespace (haven't thought about this in depth)

Also it seems we should use `or` in some way because if 'Describe high-level what changed' is found in any circumstance, it's likely to mean the template wasn't filled out.

Reply:
How about checking `[0-9A-Za-z]{3,}\\s+[0-9A-Za-z]{3,}\\s+[0-9A-Za-z]{3,}` for basically that it has at least 3 words. Clearly that still sucks for a description. Basically just making sure they:
a) do not delete the default one (so it's blank)
b) do not leave the boiler plate description